<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nicolas GOUTTE &lt;goutte@kde.org&gt;</author>
 <comment>Import dialog for Words's plain text import filter</comment>
 <class>ImportDialogUI</class>
 <widget class="QWidget" name="ImportDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plain Text Import Dialog</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="buttonGroupParagraph">
     <property name="title">
      <string>End of Paragraph</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radioParagraphAsIs">
        <property name="text">
         <string>&amp;As is: At the end of line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioParagraphSentence">
        <property name="text">
         <string>&amp;Sentence: If the end of line is the end of a sentence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioParagraphEmptyLine">
        <property name="text">
         <string>&amp;Empty Line: If the line is empty</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>KComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
